### ISSUE TITLE
Feature Request - Maintaining opened subscriptions tabs

### ISSUE THREAD

**Issue by nicolaMaza** [kamilfb/mqtt-spy#87]
Every time i open the application i lose the topic that i was monitoring.
I think it is usefull to keep the opened subscriptions tab

Nicola

**Comment by kamilfb** [comment]
Hi - this should be already available! :)

Edit your connection, go to Subscriptions tab, into Topics, Add a new topic, and tick the Create tab. Then when you open the connection it will open tabs with those topics.

You can also do it by right click on the "All" tab in your live connection, and then Configuration, Add...

Let me know if that worked.

Kamil

**Comment by nicolaMaza** [comment]
Yes, it works well, but honestly, I would never be able to find it.
It could be useful a button to save all the opened tabs (near 'new' for example)

Nicola
